server/src/models/Playlist.js

```diff
@@ -0,0 +1,27 @@
+import mongoose from 'mongoose';
+import { songSchema } from './Song';
+
+const playlistSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
```

Comment:
playlist id ? to uniquely identify playlist

Reply:
mongo assigns a _id to every schema automatically. I'll use the toString() method of type ObjectId to convert _id to a string.